database/migrations/2018_12_18_192307_journal_table.php

```diff
@@ -19,7 +19,7 @@ public function up()
             $table->text('text');
             $table->text('plan_tomorrow');
             $table->string('goal_tomorrow',255);
-            $table->boolean('goal_status');
+            $table->boolean('goal_status')->nullable();
```

Comment:
Please don't change existing migrations. Create a new one, or handle the default value in the store() method.